### PR TITLE
SOW-25: custom 404

### PR DESCRIPTION
**Pull request by Ro4052**
- Add a custom 404 page so we can direct back to the main page
- Polish page headings
- Repoint `"#"` links from the main collection to so they go to the 404 page

### CODE REVIEW

File: app/views/404.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+<div class="govuk-grid-row not-found-container">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">Page not found</h1>
+    <p class="govuk-body">
+      There is no page at
+      <strong id="pathname">{{ path }}</strong>
+    </p>
+    <p class="govuk-body">
+      This is application is for local plans draft content. You can go to
```
Comment (vferraro-scottlogic) [inline app/views/404.html]: typo here "This application is.."

**Review by vferraro-scottlogic — COMMENTED** [review]
Looks good to me. just 1 comment about a typo

File: app/views/404.html (hunk 1 above)
Comment (Ro4052) [reply app/views/404.html]: Good spot :)

Comment (Ro4052) [reply app/views/404.html]: Done!

**Review by vferraro-scottlogic — APPROVED** [review]

**Pull request merged**